py_feels/binary/time_series.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jan 20 14:50:21 2018
+
+Pulls feels and tweet times from the bit_feels database and performs
+a time-window average of the sentiments for plotting. pushes means,
+standard devaitions, classifier names and bin times into the 'stats'
+table of bit_feels. Takes two arguments: window and env
+
+window is the width of the time-bin in hours,
+env is "dev" or "prod"
+
+@author: rcehemann
+"""
+
+import pandas as pd
+from numpy import linspace, digitize, mean, std
+from datetime import datetime
+from sys import argv
+from sqlalchemy import create_engine
+
+# check environment, parse arguments
+if len(argv) < 2:
+    window = 48.
```

Comment:
Do you think we'll change the window enough to warrant it as an arg?

Reply:
my thought was that once we have streaming functionality, we may want to be able to change the window size on the fly (a la cryptowatch candle plots)

Reply:
Gotcha, that sounds good but lets not add it until we need it.